pkg/gce-pd-csi-driver/controller.go

```diff
@@ -328,11 +343,40 @@ func (gceCS *GCEControllerServer) createVolumeInternal(ctx context.Context, req
 	if err != nil {
 		return nil, status.Errorf(codes.InvalidArgument, "failed to extract parameters: %v", err.Error())
 	}
-	// Validate multiwriter
+	// https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume
+	// mutable_parameters MUST take precedence over the values from parameters.
+	mutableParams := req.GetMutableParameters()
+	// If the disk type is pd-balanced, pd-ssd, pd-standard except for pd-extreme,
+	// the IOPS and Throughput parameters are ignored.
```

Comment:
It seems like this comment is incorrect. If a disk type is pd-balanced/pd-standard etc, we'll get an error if the IOPS is set. If the throughput is set, we'll get a warning.